rpc/transport/ws/server.go

```diff
@@ -46,6 +47,10 @@ func NewWebSocketTransportAsServer(port string) (*serverWebSocketTransport, erro
 
 	serveMux.HandleFunc(apiVersionPath, wsc.request)
 	serveMux.HandleFunc(path.Join(apiVersionPath, "subscribe"), wsc.subscribe)
+
+	// Serve static site
+	serveMux.Handle("/", http.FileServer(http.FS(staticSite)))
```

Comment:
What happens if `staticSite` is empty/nil? Should we check so we don't bother trying to serve a static site if it doesn't exist?

Reply:
Good shout: [d130638](https://github.com/statechannels/go-nitro/pull/1449/commits/d130638575713581a4f844289f2111d277a86a21)